### PR TITLE
Unscaped html for error msg

### PR DESCRIPTION
**Pull request by gezichenshan**
Adding a schema property named errorUnescaped to fulfill the situation that the severside return a string with unescaped html.
see: https://jsfiddle.net/7of916d4/2/

### CODE REVIEW

**Comment by icebob** [comment]
Is it important to create a new field property? Can we use always `v-html` to show the error messages?

**Comment by gezichenshan** [comment]
I think you are right. But I've no idea whether this always v-html will result in some XSS attack?

**Comment by icebob** [comment]
I think XSS is not relevant here because we don't render user input texts, just error messages what comes from the developer. What do you think?

**Comment by gezichenshan** [comment]
Yep, I agree with you. So I'll remove the field property and add `v-html` on the error part.

**Comment by icebob** [comment]
Ok :+1:

File: src/formGenerator.vue [hunk 1]
```diff
@@ -14,8 +14,7 @@ div.vue-form-generator(v-if='schema != null')
 						button(v-for='btn in field.buttons', @click='buttonClickHandler(btn, field, $event)', :class='btn.classes') {{ btn.label }}
 				.hint(v-if='field.hint') {{ fieldHint(field) }}
 				.errors.help-block(v-if='fieldErrors(field).length > 0')
-					span(v-for='(error, index) in fieldErrors(field)', v-if='field.errorUnescaped', v-html="error", track-by='index')
-					span(v-for='(error, index) in fieldErrors(field)', v-if='!field.errorUnescaped', track-by='index') {{ error }}
+				span(v-for='(error, index) in fieldErrors(field)', v-html="error", track-by='index')
```
Comment (icebob) [inline src/formGenerator.vue]: Add indent

**Comment by icebob** [comment]
Thanks!

**Comment by gezichenshan** [comment]
With pleasure.

**Pull request merged**